code/__DEFINES/modular_skyrat/medical_defines.dm

```diff
@@ -0,0 +1,3 @@
+#define PAINKILLER_MINERSSALVE	"PAINKILLER - MINERS SALVE"
+#define PAINKILLER_MORPHINE 	"PAINKILLER - MORPHINE"
```

Comment:
why is this in this file.

Reply:
because I was told to define it in a separate file for modularisations sake.